sources/PTYTextView.m

```diff
@@ -1546,7 +1546,7 @@ - (void)rightMouseDragged:(NSEvent *)event
     [super rightMouseDragged:event];
 }
 
-- (BOOL)scrollWheelShouldSendArrowForEvent:(NSEvent *)event at:(NSPoint)point {
+- (BOOL)scrollWheelShouldSendAuxForEvent:(NSEvent *)event at:(NSPoint)point upData:(NSData**)upData downData:(NSData**)downData  {
```

Comment:
Let's separate computing the data to send from determining if data should be sent. That means two methods:

````
- (BOOL)scrollWheelShouldSendDataForEvent:(NSEvent *)event
````
(by not taking a `point` we're doing a touch of cleanup here since the point can be computed from the `event` and the caller doesn't need it for anything but passing to this method)

and
````
- (NSData *)dataToSendForScrollEvent:(NSEvent *)event
````
which calls `enclosingScrollView` and `accumulateVerticalScrollFromEvent`, and then returns the appropriate data.